hypertrace-view-generator/hypertrace-view-creator/src/main/resources/configs/span-event-view/application.conf

```diff
@@ -17,7 +17,7 @@ pinot.controllerHost = pinot-controller
 pinot.controllerPort = 9000
 pinot.timeColumn = start_time_millis
 pinot.timeUnit = MILLISECONDS
-pinot.dimensionColumns = [tenant_id, span_id, span_kind, parent_span_id, trace_id, service_id, api_id, api_name, entry_api_id, protocol_name, tags__KEYS, tags__VALUES, status_code, start_time_millis, end_time_millis, duration_millis, api_trace_id, service_name, api_boundary_type, event_name, status_message, status, api_trace_count, display_entity_name, display_span_name, request_url, error_count, api_discovery_state, exception_count, space_ids]
+pinot.dimensionColumns = [tenant_id, span_id, span_kind, parent_span_id, trace_id, service_id, api_id, api_name, entry_api_id, protocol_name, tags__KEYS, tags__VALUES, status_code, start_time_millis, end_time_millis, duration_millis, api_trace_id, service_name, api_boundary_type, event_name, status_message, status, api_trace_count, display_entity_name, display_span_name, request_url, error_count, api_discovery_state, exception_count, space_ids, api_exit_calls]
```

Comment:
@rish691 Can you add it here as well - `https://github.com/hypertrace/pinot/blob/main/pinot-servicemanager/schemas/spanEventView-schemaFile.json` ? 

This will need in docker-compose setup.

Reply:
https://github.com/hypertrace/pinot/pull/67

Reply:
oh! I see it is already added as part of this PR - https://github.com/hypertrace/pinot/pull/67/